.circleci/config.yml

```diff
@@ -298,13 +298,16 @@ commands:
 
 jobs:
   secrets-check:
-    executor: machine-executor
+    executor: docker-executor
     branches:
     steps:
       - checkout
       - run:
-          name: Scan repository for committed secrets.
+          name: Scan repository for secrets in the commit history.
```

Comment:
Are these name/description fields supposed to switched for the commands run?

Reply:
I think they just can't have the same name, although does `git secrets` scan the commit history or does trufflehog do that? It may help to add a description with more details of the distinction between the two checks so it is clear why we are doing both.